### PR TITLE
[CLOUD-2728] Integrate Microprofile OpenTracing

### PR DESCRIPTION
**Pull request by bstansberry**
Signed-off-by: Brian Stansberry <brian.stansberry@redhat.com>

https://issues.jboss.org/browse/CLOUD-2728

### CODE REVIEW

File: jboss-eap-config-tracing/module.yaml [hunk 1]
```diff
@@ -0,0 +1,17 @@
...
+description: EAP OpenTracing configuration scripts
+execute:
+- script: configure.sh
+  user: '185'
+envs:
+    - name: "WILDFLY_TRACING_ENABLED"
+      example: "true"
+      description: "Use the microprofile-tracing-smallrye subsystem to add OpenTracing support."
+    - name: "JAEGER_AGENT_HOST"
+      example: "true"
```
Comment (maschmid) [inline jboss-eap-config-tracing/module.yaml]: "true" does not make sense as en example here.

Comment (bstansberry) [reply jboss-eap-config-tracing/module.yaml]: Oops; thanks, will fix.

Comment (bstansberry) [reply jboss-eap-config-tracing/module.yaml]: I replaced this with an example hostname from a Service that exposes a jaeger agent.

**Comment by bstansberry** [comment]
I'll fix the standalone-openshift.xml conflict, which is just because the mod_cluster subsystem, which immediately follows the newly added marker tag, has changed. Resolving this won't change any functionality.

**Pull request merged**